### PR TITLE
Add support for scissors

### PR DESCRIPTION
**Pull request by knokko**
Like I said in that issue, I believe it would be nice to add scissor support to golem. So, here is a pull request for it.

### CODE REVIEW

**Comment by knokko** [comment]
Hm... some tests are failing, but the complaints do not seem to be related to this pull request. Is this something that you or me need to solve first? Or can you ignore them and merge anyway?

**Comment by lenscas** [comment]
the macos error is caused by the version of the github action used to get rust.

Update it and that should be fixed.

The clippy error needs a code change though :)

**Comment by knokko** [comment]
Ok, I cleaned up some code and bumped the version like you told me. That worked indeed.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 /target
 **/*.rs.bk
 Cargo.lock
+/.idea/
```
Comment (elliegoldstein) [inline .gitignore:4]: Did you mean to commit this?

**Review by elliegoldstein — APPROVED** [review]
Looks good to me, just one comment.
(Thanks for the code cleanup btw!)

File: .gitignore (hunk 1 above)
Comment (knokko) [reply .gitignore:4]: Yes. The .idea folder contains mostly internal Intellij config files that I would rather not commit. Adding it to .gitignore makes sure that I (or anyone in the future) won't commit these files by accident.

Comment (Ratysz) [reply .gitignore:4]: You can keep changes like this local by putting them into a changeset separate from the one you're committing. This way stuff specific to your setup doesn't bleed into the main repository.

**Pull request merged**